### ISSUE TITLE
*lookup behavior is inconsistent, broken

### ISSUE THREAD

**Issue by paul-pearce** [zmap/zdns#110]
Currently:

```
$ echo "www.zdns-testing.com" | ./zdns/zdns alookup | jq "."
{
  "timestamp": "2017-08-20T15:22:26-07:00",
  "status": "NO_ANSWER",
  "class": "IN",
  "name": "www.zdns-testing.com"
}

echo "www.zdns-testing.com" | ./zdns/zdns alookup --ipv4-lookup | jq "."
{
  "data": {
    "ipv4_addresses": [
      "1.2.3.4",
      "2.3.4.5",
      "3.4.5.6"
    ]
  },
  "timestamp": "2017-08-20T15:21:04-07:00",
  "status": "NOERROR",
  "class": "IN",
  "name": "www.zdns-testing.com"
}
```

This is because we explicitly force this condition, here: https://github.com/zmap/zdns/blob/master/modules/alookup/alookup.go#L110

However, for mxlookup, we do not:

```
$ echo "www.zdns-testing.com" | ./zdns/zdns mxlookup --ipv4-lookup | jq "."
{
  "data": {
    "exchanges": [
      {
        "ttl": 299,
        "ipv4_addresses": [
          "1.2.3.4",
          "2.3.4.5"
        ],
        "preference": 1,
        "class": "IN",
        "type": "MX",
        "name": "mx1.zdns-testing.com"
      },
      {
        "ttl": 299,
        "ipv4_addresses": [
          "5.6.7.8"
        ],
        "preference": 5,
        "class": "IN",
        "type": "MX",
        "name": "mx2.zdns-testing.com"
      },
      {
        "ttl": 299,
        "preference": 10,
        "class": "IN",
        "type": "MX",
        "name": "mx1.censys.io"
      }
    ]
  },
  "timestamp": "2017-08-20T15:23:29-07:00",
  "status": "NOERROR",
  "class": "IN",
  "name": "www.zdns-testing.com"
}
$ echo "www.zdns-testing.com" | ./zdns/zdns mxlookup | jq "."
{
  "data": {
    "exchanges": [
      {
        "ttl": 299,
        "preference": 1,
        "class": "IN",
        "type": "MX",
        "name": "mx1.zdns-testing.com"
      },
      {
        "ttl": 299,
        "preference": 5,
        "class": "IN",
        "type": "MX",
        "name": "mx2.zdns-testing.com"
      },
      {
        "ttl": 299,
        "preference": 10,
        "class": "IN",
        "type": "MX",
        "name": "mx1.censys.io"
      }
    ]
  },
  "timestamp": "2017-08-20T15:23:45-07:00",
  "status": "NOERROR",
  "class": "IN",
  "name": "www.zdns-testing.com"
}
```

If we remove the condition at https://github.com/zmap/zdns/blob/master/modules/alookup/alookup.go#L110, we get:

```
$ echo "www.zdns-testing.com" | ./zdns/zdns alookup | jq "."
{
  "data": {},
  "timestamp": "2017-08-20T15:25:14-07:00",
  "status": "NOERROR",
  "class": "IN",
  "name": "www.zdns-testing.com"
}
```

This raises a number of questions:

1. Should we default to --ipv4-lookup, if no option is supplied?
2. If no, what should the default invocation behavior be?
3. Should the basic output structure of mxlookup and alookup be the same? Currently mxlookup includes _significantly_ more info than alookup.

**Comment by zakird** [comment]
I'm fine with having `--ipv4-lookup` be default. The only awkward thing is then: what if you only want IPv6? How do you get rid of IPv4 lookup? It seems a awkward to have defaults like that. I'm not fundamentally opposed. It's just something we should think about.

**Comment by paul-pearce** [comment]
What I imagined is a check where if both are false (after parsing), we overrride the ipv4==false with true.

**Comment by zakird** [comment]
That seems fine.

**Comment by paul-pearce** [comment]
Resolved by #113
